### ISSUE TITLE
Fix -c opt builds on OS X [phab:D896]

### ISSUE THREAD

**Issue by kythe** [google/kythe#1205]
_Created by **zarko** at 2016-05-17 00:13:02_: 

Without this fix, you'll see:
```
ld: warning: option -s is obsolete and being ignored
ld: internal error: atom not found in symbolIndex(__ZN6snappy18SnappyDecompressor17DecompressAllTagsINS_17SnappyArrayWriterEEEvPT_) for architecture x86_64
clang: error: linker command failed with exit code 1 (use -v to see invocation)
```

**Comment by kythe** [comment]
_**schroederc** accepted this revision at 2016-05-17 04:00:24_

**Comment by kythe** [comment]
_**zarko** closed via commit 2c4992a2166fe66575c155eac18e5602dac94a0c at 2016-05-17 17:48:31_
